binder/environment.yml

```diff
@@ -8,5 +8,6 @@ dependencies:
   - notebook
   - pip
   - python>=3.9,<3.11
+  - unitpackage==0.8.0
   - pip:
-    - unitpackage==0.8.0
+    - bash_kernel
```

Comment:
Do we need the `bash_kernel` package?

Reply:
seems to work without it. Not sure what prevented a proper build in previous runs